index.js

```diff
@@ -465,11 +420,11 @@ export function StateT(M) {
   //. Z.of (Maybe, 2)
   //. ```
   StateT.prototype['fantasy-land/ap'] = function(mf) {
-    var mx = this;
-    return new StateT (function(state) {
-      var get = StateT.evalState (state); // Monad {st, val}
+    const mx = this;
```

Comment:
This is no longer strictly necessary, although, since it also clarifies the role of `this`, you may want to keep it.

Reply:
I think I am going to keep it.